MainModule/Server/Core/HTTP.lua

```diff
@@ -38,7 +38,11 @@ return function(Vargs)
 	server.HTTP = {
 		Init = Init;
 		Service = service.HttpService;
+<<<<<<< Updated upstream
```

Comment:
What's this?

Seems like you added some jargon to your file.

Everything else is great. Good job.